ui/app/AppLayouts/Communities/popups/TransferOwnershipPopup.qml

```diff
@@ -109,9 +110,10 @@ StatusDialog {
 
                 onClicked: {
                     // Pre-populated dialog with the relevant Owner token info:
-                    root.sendModalPopup.preSelectedSendType = Constants.SendType.Transfer
+                    root.sendModalPopup.preSelectedSendType = Constants.SendType.ERC721Transfer
                     root.sendModalPopup.preSelectedAccount = ModelUtils.getByKey(root.accounts, "address", token.accountAddress)
-                    root.sendModalPopup.preSelectedHoldingID = token.key
+                    const uid = token.chainId+"+"+token.tokenAddress.toLowerCase()+"+"+"0" // TODO use getUidForData
```

Comment:
Have you tried all the flows where the popup is open just to be sure the new introduced `tokenAddress` is set in all cases?
- `OverviewSettingsPanel` --> I do see changes in here
- `MintTokensSettingsPanel` -->  I do see changes in here
- `WalletLayout` --> I see a potential issue in here: 

```
 Global.openTransferOwnershipPopup(walletStore.currentViewedCollectible.communityId,
                                                      tokenItem.communityName,
                                                      tokenItem.communityImage,
                                                      {
                                                          "key": walletStore.currentViewedHoldingID,
                                                          "privilegesLevel": tokenItem.communityPrivilegesLevel,
                                                          "chainId": tokenItem.chainId,
                                                          "name": tokenItem.name,
                                                          "artworkSource": tokenItem.artworkSource,
                                                          "accountAddress": tokenItem.contractAddress
                                                      },
                                                      walletStore.accounts,
                                                      root.sendModalPopup)
```
There's now `tokenAddress` property set.. probably it should be this `accountAddress` that is assigned to `tokenItem.contractAddress`.

Reply:
Yes, I fixed properties set in WalletLayout.